test/Compiler.test.js

```diff
@@ -181,24 +181,6 @@ describe("Compiler", () => {
 				}
 			});
 		});
-		describe("parser", () => {
-			describe("plugin", () => {
-				it("invokes sets a 'compilation' plugin", (done) => {
-					compiler.plugin = sinon.spy();
-					compiler.parser.plugin();
```

Comment:
These 2 cases test deprecated methods. Should I keep them?

Reply:
They test the compat layer. We can remove the compat layer here anyway in webpack 4.